packages/kbn-es/src/serverless_resources/roles.yml

```diff
@@ -387,6 +387,7 @@ soc_manager:
         - feature_actions.all
         - feature_builtInAlerts.all
         - feature_osquery.all
+        - feature_indexPatterns.all # Detections Data Views
```

Comment:
Hey. I think you missed my prior comment. can you update line 8 of this file with this path instead:

```
x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless/es_serverless_resources/roles.yml
```

Reply:
crap. never mind
this is the Kibana PR 🤦